cmd/translate/main.go

```diff
@@ -84,6 +125,7 @@ func main() {
 func init() {
 	cobra.OnInitialize(initConfig)
 	rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is ./translate.yaml)")
+	rootCmd.PersistentFlags().StringVar(&envFile, "env", "", "environment file")
```

Comment:
We can remove this flag, we'll never load env variable files in the **translate** service.

There are other ways to consume env variable files. E.g. when running tests in the VS Code, we can let go extension know where to find the env file with configuration `"go.testEnvFile": "${workspaceFolder}/.env.test"`. And so on.